src/ucp/api/ucp.h

```diff
@@ -1223,6 +1224,12 @@ typedef struct {
      * Pointer to user data passed to callback function.
      */
     void          *user_data;
+
+    /**
+     * Reply buffer. Can be used for storing operation result, like in
+     * @ref ucp_atomic_fetch_nbx.
```

Comment:
` Reply buffer. Can be used for storing operation result, for example by @ref ucp_atomic_fetch_nbx.`